database/sproc/group.sql

```diff
@@ -54,3 +56,24 @@ END$$
 
 DELIMITER ;
 
+DELIMITER $$
+USE `doughBros_db`$$
+CREATE PROCEDURE `getGroupsByUser`(IN `user_id` INT(8))
```

Comment:
user_id is a Firebase UID now, so it is no longer an integer! make sure this is the same type as the column defined in "user" table

Reply:
These changes and sprocs have been fixed in another branch, I think this PR can be closed